All translations updated to current progress

Ben21821 opened the pull request.

yasd2 commented:
Hey Ben! I will manually include the translations in MCCallouts, because I don't want to change the english files in here. I have fixed a simple bracket mistake in BicycleAccident.xml.
Do the translations work for you? Because for me it only shows empty rectangles, maybe that's because I have a different language?

Ben21821 commented:
> 嘿，Ben！我会手动把翻译添加到 MCCallouts 中，因为我不想修改这里的英文文件。我已经修复了 BicycleAccident.xml 中一个简单的括号错误。 翻译对你有用吗？因为我的翻译只显示空的矩形，可能是因为我的语言不一样吧？

Ben21821 commented:
> Hey Ben! I will manually include the translations in MCCallouts, because I don't want to change the english files in here. I have fixed a simple bracket mistake in BicycleAccident.xml. Do the translations work for you? Because for me it only shows empty rectangles, maybe that's because I have a different language?

Sorry, maybe I accidentally deleted it in the translation, I have fixed it

yasd2 commented:
Do the translations work for you? Because for me it only shows empty rectangles, maybe that's because I have a different language?

Ben21821 commented:
I haven't tested the translation file yet because it's not ready after my game update. As far as I know, Chinese encoding should require UTF-8 to be displayed properly.

yasd2 commented:
Yes, I am familiar with that and it's already supported ^^
Just let me know if it works for you as soon as your game is updated :)

Ben21821 commented:
Ok, I'll try to restore the game as soon as possible, you know every update is a headache  :-C

Ben21821 commented:
The language file can be used normally, but the characters displayed in the location information are garbled，
![Uploading 捕获.PNG…]()

yasd2 commented:
Unforunately the link doesn't work, do you have discord? There it might be easier to chat :)

Ben21821 commented:
I will contact you on discord
